_includes/sidebar.html

```diff
@@ -3,10 +3,12 @@
   <li class="sidebar-title">Get started</li>
 
     <ul class="sidebar-items">
-      <li><a href="/setup/">Install</a></li>
-      <li><a href="/getting-started/">Create and run your first app</a></li>
-      <li><a href="/ide-setup/">Setting up the IDE</a></li>
-      <li><a href="/bootstrap-into-dart/">Bootstrap into Dart</a></li>
+      <li><a href="/get-started/">1: Overview</a></li>
+      <li><a href="/get-started/install/">2: Install Flutter</a></li>
+      <li><a href="/get-started/editor-configuration/">3: Editor configuration</a></li>
```

Comment:
"Configure editor". It's weird to have a noun when other steps are verbs. Verbs are better.

Reply:
Done